src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  width: 100%;
+  background-color: #f8f8f8;
+}
+
+.nav-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 20px;
+}
+
+.logo {
+  display: block;
+}
+
+nav ul {
+  display: flex;
+  list-style: none;
+}
+
+nav ul li {
+  margin: 0 15px;
+}
+
+nav ul li:last-child {
+  margin-right: 0;
+}
+
+nav ul li:first-child {
+  margin-left: 0;
+}
+
+nav a {
+  text-decoration: none;
+  color: #000;
+  display: block;
+  text-transform: uppercase;
+  text-align: center;
+  line-height: 60px;
+  position: relative;
+}
+
+nav a:hover {
+  color: var(--blue);
```

Comment:
The CSS variable '--blue' is used here, but it is defined after this line in the ':root' selector. While CSS variables are globally scoped and this will not cause an error, it's a good practice to define variables before they are used to improve readability and maintainability.